shared/constants/fs.js

```diff
@@ -0,0 +1,7 @@
+// @flow
+import * as I from 'immutable'
+import * as Types from './types/fs'
+
+export const makeState: I.RecordFactory<Types._State> = I.Record({
+  counter: 0,
```

Comment:
Is this where we'll populate default states for the redux store?

Reply:
Sort of. Really this happens in the reducer. It just so happens to take this value. The reducer is called with undefined as the initial state